Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/ObjectBinding.Web.Development.WebTesting.TestSite.NetFramework.csproj

```diff
@@ -1,231 +1,28 @@
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\011ace7d-5149-4117-8ace-dd33d80b8f96" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\19214979-d614-4afa-b074-7b05fe37ea65" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\39aab7cd-9916-4549-a8ec-145fc9cfae2e" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\6866ca48-8957-4f26-ae5f-78a3f6dcc4de" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\88ca0bf3-2a08-4d93-bee7-0454052a922d" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\9e713934-1226-4669-880e-c07c22cdab19" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\a2752869-e46b-4cfa-b89f-0b824e42b250" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\a97d84b0-c1c9-4580-a6c1-1fed1ee8c041" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\c8ace752-55f6-4074-8890-130276ea6cd1" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\c93f1935-2d55-4b2a-9fd5-219739324cfb" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\eb94bfdb-1140-46f8-971f-e4b41dae13b8" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Person\ef898bb1-7095-46d2-872b-5f732a7c0036" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Job\02e98644-2051-464d-96f5-0d778862a260" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Job\f5758e93-4eb6-457c-abea-038322f986cd" />
-    <Content Include="objects\Remotion.ObjectBinding.Sample.Job\62187fba-be7e-4a6e-8a0c-7e3f1b696d7f" />
-    <Content Include="..\..\Remotion LGPL.licenseheader">
-      <Link>Remotion LGPL.licenseheader</Link>
-    </Content>
-    <Content Include="index.html" />
-    <Content Include="packages.config" />
-    <None Include="Web.Debug.config">
-      <DependentUpon>Web.config</DependentUpon>
-    </None>
-    <None Include="Web.Release.config">
-      <DependentUpon>Web.config</DependentUpon>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Global.asax" />
-    <Content Include="Web.config" />
+    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
+    <ProjectReference Include="..\..\Development\Web\Development.Web.csproj" />
+    <ProjectReference Include="..\..\Web\Core\Web.Core.csproj" />
+    <ProjectReference Include="..\..\Web\Development.WebTesting.TestSite.Infrastructure\Web.Development.WebTesting.TestSite.Infrastructure.csproj" />
+    <ProjectReference Include="..\Core\ObjectBinding.Core.csproj" />
+    <ProjectReference Include="..\Sample\ObjectBinding.Sample.csproj" />
+    <ProjectReference Include="..\Web.Development.WebTesting.TestSite.Shared\ObjectBinding.Web.Development.WebTesting.TestSite.Shared.csproj" />
+    <ProjectReference Include="..\Web.UnitTests\ObjectBinding.Web.UnitTests.csproj" />
+    <ProjectReference Include="..\Web\ObjectBinding.Web.csproj" />
```

Comment:
I think we could probably drop sevreal of the project refences thanks to transitive refs?

Reply:
Transitive references don't seem to work with .NET Framework so we still have to reference most of them. While there are a few "unused" references per project I would keep them in as well as somebody might depend on them at runtime. On develop, we convert these projects to .NET soon anyway. So I would leave them as is for compatibility in the backports.